src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/hyjalAI.cpp

```diff
@@ -831,7 +831,10 @@ void hyjalAI::UpdateAI(uint32 diff)
                     }
                     else if (BossGUID[i] == BossGUID[1])
                     {
-                        Talk(SUCCESS);
+                        if (me->GetEntry()!=JAINA) // jaina yell success after select gossip
```

Comment:
Contradiction here
```cpp
// jaina yell success after select gossip
```
but
```txt
!= JAINA
```

Reply:
Accoding to the reference video,Jaina yell success after select gossip,not yell after reach waypoint.So I move the behavior from here.

Reply:
You force here any creature except Jaina to yell, take a closer look at the check

Reply:
I made a mistake.Updated code for this.